src/Presentation/TabsHtmlBuilder.php

```diff
@@ -6,22 +6,24 @@
 
 use MediaWiki\Html\TemplateParser;
 use MediaWiki\Title\Title;
+use MediaWiki\Title\TitleFactory;
 use ProfessionalWiki\WikibaseFacetedSearch\Application\Config;
+use ProfessionalWiki\WikibaseFacetedSearch\Application\ConfigAuthorizer;
 use ProfessionalWiki\WikibaseFacetedSearch\Application\ItemTypeLabelLookup;
 use ProfessionalWiki\WikibaseFacetedSearch\Application\Query;
 use ProfessionalWiki\WikibaseFacetedSearch\Application\QueryStringParser;
 use ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension;
 use Wikibase\DataModel\Entity\ItemId;
-use MediaWiki\User\User;
 
 class TabsHtmlBuilder {
 
 	public function __construct(
 		private readonly Config $config,
+		private readonly ConfigAuthorizer $configAuthorizer,
 		private readonly ItemTypeLabelLookup $itemTypeLabelLookup,
 		private readonly TemplateParser $templateParser,
-		private readonly QueryStringParser $queryStringParser,
-		private readonly User $user,
+		private readonly TitleFactory $titleFactory,
+		private readonly QueryStringParser $queryStringParser
```

Comment:
Did you arrange these alphabetically? Usually we don't do that, and instead just add new collaborators at the end.